cypress/integration/account_settings/sidebar/channel_switcher_spec.js

```diff
@@ -94,4 +96,73 @@ describe('Account Settings > Sidebar > Channel Switcher', () => {
         // * Channel Switcher button should appear at the bottom of the left-hand-side bar
         cy.get('#sidebarSwitcherButton').should('be.visible');
     });
+
+    it('set channel switcher setting to On and test on click of sidebar switcher button', () => {
+        // 1. Go to Account Settings modal > Sidebar > Channel Switcher and set setting to On
+        cy.toAccountSettingsModalChannelSwitcher('user-1');
+
+        // 2. Go to a known team and channel
+        cy.visit('/ad-1/channels/town-square');
+        cy.get('#channelHeaderTitle').should('be.visible').should('contain', 'Town Square');
+
+        // 3. Click the sidebar switcher button
+        cy.get('#sidebarSwitcherButton').click();
+
+        // * Channel switcher hint should be visible
```

Comment:
What's the difference in meaning between `N.` and `*` in the comments?

Reply:
Those are test step and assertion.  You'll find that on top of each file as general comments/reminder when writing a test case.

```
// - [number] indicates a test step (e.g. 1. Go to a page)
// - [*] indicates an assertion (e.g. * Check the title)
// - Use element ID when selecting an element. Create one if none.
```